orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/MessageHandler.kt

```diff
@@ -37,6 +38,11 @@ interface MessageHandler<M : Message> : (Message) -> Unit {
       throw IllegalArgumentException("Unsupported message type ${message.javaClass.simpleName}")
     }
 
+  fun Collection<ExceptionHandler>.shouldRetry(ex: Exception, taskName: String?): ExceptionHandler.Response? {
```

Comment:
I had to change it this way because of some failing tests - I think somewhere is using task as a mock and doesn't have a name?